ksqldb-engine/src/test/java/io/confluent/ksql/engine/RuntimeAssignorTest.java

```diff
@@ -30,8 +29,8 @@ public class RuntimeAssignorTest {
   public BinPackedPersistentQueryMetadataImpl queryMetadata;
   private final QueryId query1 = new QueryId("Test1");
   private final QueryId query2 = new QueryId("Test2");
-  private final Collection<SourceName> sources1 = Collections.singleton(SourceName.of("test1"));
-  private final Collection<SourceName> sources2 = Collections.singleton(SourceName.of("test2"));
+  private final Collection<String> sources1 = Collections.singleton("test1");
```

Comment:
Ah interesting, the unit test is no longer interested in sources at all, since this coverage has changed to use different topics instead of sources. Maybe we should rename the tests from `shouldAddingAQueryWithSameSourcesWillAddARuntime()` to `shouldAddingAQueryWithSameTopicsWillAddARuntime()`, and similarly for the others too?

Also, is there anywhere to actually add test coverage that two sources that share the same topic will create new runtimes? (Your judgment call on whether this is actually necessary or not. Seems nice to have if it's not difficult, though.)

Reply:
Ah good point. I will add a unit test for the runtime assignor. It _shouldn't_ matter but better to have.